drivers/storage/portworx/util/util.go

```diff
@@ -400,10 +401,21 @@ func IsVsphere(cluster *corev1.StorageCluster) bool {
 	return false
 }
 
+// IsPure returns true if PURE_SAN_TYPE  is present in the spec
+func IsPure(cluster *corev1.StorageCluster) bool {
+	envValue, exists := GetClusterEnvValue(cluster, "PURE_FLASHARRAY_SAN_TYPE")
+	if exists && len(envValue) > 0 {
+		return true
```

Comment:
Will there be any valid/invalid value for PURE_FLASHARRAY_SAN_TYPE?

Reply:
This is FA transport type and its same as IsVsphere check, we only check if env var is present or not, value is not validated here.